src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/CLLComponent.java

```diff
@@ -0,0 +1,20 @@
+package com.synopsys.integration.detect.workflow.componentlocationanalysis;
+
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+
+public class CLLComponent {
```

Comment:
We should fully qualified names. Here, Component is sufficient. The package name (should be renamed to component.location.) provides sufficient context.